actions/mongodb_explain_action_test.go

```diff
@@ -19,11 +19,17 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
 	"testing"
 
+	"github.com/Masterminds/semver"
```

Comment:
We have our own package for that: https://github.com/percona/pmm/blob/PMM-2.0/version/parsed.go

Example of usage:
* https://github.com/percona/pmm-managed/blob/22524d05feea287a2d315df3e5c2f557242ef9e4/services/checks/checks.go#L74-L79
* https://github.com/percona/pmm-managed/blob/22524d05feea287a2d315df3e5c2f557242ef9e4/services/checks/checks.go#L640